### PR TITLE
"Now Playing" added

### PR DESCRIPTION
**Pull request by HeyImKyu**
- Added "Now Playing"
- Added KanaConverter
  - Now latinizing, then converting kana.
  - All chars that are still unknown after that are replaced with *
- If EndTime is 0, return no time
  - Happens e.g. with YouTube

### CODE REVIEW

**Comment by HeyImKyu** [comment]
Using an edited version of [pilotMike/KanaConverter](https://github.com/pilotMike/KanaConverter). ^^

File: ViewModel/OSCHandler.cs [hunk 1]
```diff
@@ -1,132 +1,139 @@
...

                     case DisplayTypes.CycleMultiple:
                         foreach (string text in chunks)
                             await DoSend(VRCSTTViewModelFactory.GetInstance().SecondsTimer, text, false);
+                        break;

+                    case DisplayTypes.NowPlaying:
+                        var musicInfo = await MusicHandler.GetMusicInformation();
+                        var formatted = MusicHandler.FormatInformation(musicInfo);
+
```
[the remark this answers is not in the document]
Comment (HeyImKyu) [reply ViewModel/OSCHandler.cs]: Moved in https://github.com/HeyImKyu/VRCSTT/pull/22/commits/0dcaa086d25e041831717ef2df54298346e96b6c.

File: VRCSTT.csproj [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk" ToolsVersion="Current">
   <PropertyGroup>
     <OutputType>WinExe</OutputType>
-    <TargetFramework>net6.0-windows</TargetFramework>
+    <TargetFramework>net6.0-windows10.0.19041.0</TargetFramework>
```
[the remark this answers is not in the document]
Comment (HeyImKyu) [reply VRCSTT.csproj:4]: Added test script in https://github.com/HeyImKyu/VRCSTT/pull/22/commits/79ca00e3c9b584e111744a30335f10419cbe215d.

**Review by DaitosGithub — APPROVED** [review]
╰(*°▽°*)╯

**Pull request merged**